app_test.go

```diff
@@ -975,17 +975,17 @@ func TestRequiredFlagAppRunBehavior(t *testing.T) {
 		{
 			testCase:    "valid_case_required_flag_input_on_command",
 			appRunInput: []string{"myCLI", "myCommand", "--requiredFlag", "cats"},
-			appCommands: []Command{Command{
+			appCommands: []Command{{
 				Name:  "myCommand",
 				Flags: []Flag{StringFlag{Name: "requiredFlag", Required: true}},
 			}},
 		},
 		{
 			testCase:    "valid_case_required_flag_input_on_subcommand",
 			appRunInput: []string{"myCLI", "myCommand", "mySubCommand", "--requiredFlag", "cats"},
-			appCommands: []Command{Command{
+			appCommands: []Command{{
 				Name: "myCommand",
-				Subcommands: []Command{Command{
```

Comment:
gofmt removed these, which should be fine.

Reply:
I didn't realize you could do this!!! Nice